custodia/secrets.py

```diff
@@ -31,9 +33,12 @@ def _db_container_key(self, default, trail):
         if len(trail) > 1:
             f = self._db_key(trail)
         elif len(trail) == 1 and trail[0] != '':
+            self.logger.error('Forbidden action. Trying to create an '
+                              'container but the trail: %s is not '
+                              'empty.', trail)
```

Comment:
As above talking about "trail" is obscure, even more saying it is empty. there reason we check for an empty trail is that when python tokenizes the string it will append an empty component to the list only if the original string terminates with a '/'. And this is what the error is about, container names must end in '/', so the message should say somethign along those line.